test/random_test.cpp

```diff
@@ -2,7 +2,7 @@
 
 #include "engine.h"
 
-using namespace devilution;
+namespace devilution {
```

Comment:
Why change the namespace compared to all other tests?

Reply:
I mentioned this in the comment for that commit, but this is recommended for use with the GTest framework and required if you want to use [FRIEND_TEST](http://google.github.io/googletest/reference/testing.html#FRIEND_TEST)/test fixtures for classes with private members. Given the nature of an RNG it's likely that this would occur if a class is created for the RNG implementation.